packages/components/src/ecosystem_bar/index.module.scss

```diff
@@ -0,0 +1,82 @@
+@import "~nebenan-ui-kit";
+
+// IMPORTANT: global class names dependency
+// 'client/components/layout_header/index.jsx'
```

Comment:
do we have the same component in biz and org?

Reply:
Yes, we do!